test/util/bitcoin-unit-tests.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+PROCESSES=4
```

Comment:
Would it make sense just to do `xargs ... -P 0`?

Reply:
I don't like hardcoding the parallellism here, but with `-P 0` my system will run all tests simultaneously - that's a bit scary w.r.t. resource usage.